app/views/directives/_copy-to-clipboard.html

```diff
@@ -1,9 +1,10 @@
 <div class="input-group copy-to-clipboard">
-  <input id="{{id}}" type="text" class="form-control" value="{{clipboardText}}" readonly select-on-focus>
+  <input id="{{id}}" type="text" class="form-control" value="{{clipboardText}}" ng-disabled="isDisabled" ng-readonly="!isDisabled" select-on-focus>
   <span class="input-group-btn" ng-hide="hidden">
-    <button data-clipboard-target="#{{id}}"
+    <a data-clipboard-target="#{{id}}"
+            ng-disabled="isDisabled"
```

Comment:
no it wont disable the link. It's just to graphically distinguish the widget in disabled and not-disabled state. we already spoke about it with @jwforres https://github.com/openshift/origin-web-console/pull/298#issuecomment-239510865